package.json

```diff
@@ -113,5 +111,8 @@
     "hooks": {
       "pre-commit": "lint-staged"
     }
+  },
+  "resolutions": {
```

Comment:
We could make a PR on cozy-script to avoid this resolution

Reply:
> We could make a PR on cozy-script to avoid this resolution

@cballevre that would be preferrable for sure but I'm not sure it would be very easy to update cozy-scripts to eslint 8